cdk/script/ci

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+nvm_available() {
+  type -t nvm > /dev/null
+}
+
+source_nvm() {
+  if ! nvm_available; then
+    [ -e "/usr/local/opt/nvm/nvm.sh" ] && source /usr/local/opt/nvm/nvm.sh
+  fi
+  if ! nvm_available; then
+    [ -e "$HOME/.nvm/nvm.sh" ] && source "$HOME/.nvm/nvm.sh"
+  fi
+}
+
+source_nvm
+nvm install
+nvm use
+
+npm install -g yarn
```

Comment:
As this repo doesn't have any pre-existing node projects, shall we stay on `npm` to avoid this global install?

Reply:
This caused issues on presence-indicator recently too - https://github.com/guardian/presence-indicator/pull/127.

Reply:
I have no preference here (this is just what the latest `new` command produced when I started this migration). I will double check whether Mobile team have a preference in the `yarn` vs `npm` debate. If not, I'll swap to the latter.

Reply:
cc @JamieB-gu @frankie297 @waisingyiu - do you have any preference here?

Reply:
I'd suggest npm. I think there are a few reasons it's simpler and it's the default in the docs: https://github.com/guardian/cdk/blob/main/docs/setting-up-a-gucdk-project.md

Only caveat is that the one TS project the MSS team maintain (mobile-purchases) is on Yarn - will this cause any confusion for you when switching between projects @frankie297 @waisingyiu ?

Reply:
I think it may be good to use `npm` as it is the default in the gucdk docs and we may need to reference the docs to do GuCDK upgrade going forward.  I understood that there might be a little confusion when working on `mobile-purchases`, but I reckon it may not be a big issue.

Reply:
Thanks @JamieB-gu & @waisingyiu - I've updated to use `npm` in https://github.com/guardian/mobile-save-for-later/pull/64/commits/ddbb924420d159e80f7b0d3ebe10b3befbfafb67.